### PR TITLE
BingX Trade One Click Close All Positions - bug fix

### LINKED ISSUE #6320: [BUG] BingX: trade_oneclick_close_all_positions doesn't work
**Description**
When you trigger this action, no matter which trigger provided, you can't give any paramets, since this action doesn't request one. 
Yet when you fire the action, no positions at BingX are closed and the return message just says:

_Invalid parameters, err:Key: 'APIOneClickClosePositionRequest.Symbol' Error:Field validation for 'Symbol' failed on the 'endswith' tag
Key: 'APIOneClickClosePositionRequest.PositionID' Error:Field validation for 'PositionID' failed on the 'gt' tag_

**To Reproduce**
1. Create a new Workflow 
2. Select any trigger. eg a Webhook
3. select "BingX" as action and chose trade_oneclick_close_all_positions
4. Apply your BingX Account to it
5. Make sure you have at least one open position in BingX Perpetual Futures Trading Account to be closed
6. Fire the Webhook and see the error message 
7. Positions stay open

**Workaround**
I found out, if you just use the "Generic API Call" Function and place the API Path "/api/v1/user/oneClickCloseAllPositions" in it, this will work perfectly. 
So whatever your "trace_oneclick_close_all_positions" does, it certainly doesn't, what the BingX API expects: To simpley call "/api/v1/user/oneClickCloseAllPositions" as HTTP-POST.

**Expected behavior**
The Workflow should work as designed and close all open positions at Market Price and don't return weird error messages reffering to PosionIDs you can't even provide as input parameter of the action. Like described above, my simple self-made-workaround does the job as expected.

**Additional context**
This is a very urgent thing for me since my whole autotrading strategy depends on this action to work properly. Otherwise I only could open positions automatically but never close them when they reach TP or SL Level.

Thanks in Advance.

### PR DESCRIPTION
**Pull request by michelle0927**
Resolves #6320

### CODE REVIEW

**Review by luancazarine — APPROVED** [review]
Hi @michelle0927, thanks for your contribution! LGTM!

File: components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs [hunk 1]
```diff
@@ -2,7 +2,7 @@ import bingx from "../../bingx.app.mjs";
 
 export default {
   name: "BingX Trade One Click Close All Positions",
-  version: "0.0.3",
+  version: "0.0.4",
```
Comment (dylburger) [inline components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs:5]:
```suggestion
  version: "0.0.2",
```

Comment (dylburger) [reply components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs:5]: Added a comma at the end for consistency with other lines.

File: components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs [hunk 2]
```diff
@@ -11,7 +11,7 @@ export default {
   type: "action",
   async run({ $ }) {
     const API_METHOD = "POST";
-    const API_PATH = "/api/v1/user/oneClickClosePosition";
+    const API_PATH = "/api/v1/user/oneClickCloseAllPositions";
```
[the remark this answers is not in the document]
Comment (dylburger) [reply components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs:14]: No changes necessary.

File: components/bingx/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@pipedream/bingx",
-  "version": "0.1.1",
+  "version": "0.1.2",
```
Comment (dylburger) [inline components/bingx/package.json:3]:
```suggestion
  "version": "0.1.2",
```

Comment (dylburger) [reply components/bingx/package.json:3]: No changes necessary.

File: components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs (hunk 1 above)
Comment (dylburger) [inline components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs:5]:
```suggestion
  version: "0.0.4",
```

Comment (dylburger) [reply components/bingx/actions/trade-oneclick-close-all-positions/trade-oneclick-close-all-positions.mjs:5]: No changes necessary.

**Pull request merged**